velox/exec/tests/SimpleArrayAggAggregate.cpp

```diff
@@ -157,7 +157,8 @@ exec::AggregateRegistrationResult registerSimpleArrayAggAggregate(
         return std::make_unique<SimpleAggregateAdapter<ArrayAggAggregate>>(
             resultType);
       },
-      /*registerCompanionFunctions*/ true);
+      false /*registerCompanionFunctions*/,
```

Comment:
The original registerCompanionFunctions was true, so let's keep it true. Same for SimpleAverageAggregate.cpp. I think this is what causes the red signals in CI.

Reply:
Updated, thanks. The failure on MacOS is being fixed with https://github.com/facebookincubator/velox/pull/8971, and I will rebase to it after it's landed.